### DISCUSSION TITLE
After ocr the image, how to fill the content back to the original position?

### DISCUSSION THREAD

**Discussion by ZZZ1223** [pypdfium2-team/pypdfium2#323] [Q&A] [answered]
When I use "list(page.get_objects(filter=(pdfium_c.FPDF_PAGEOBJ_IMAGE, FPDF_PAGEOBJ_TEXT))", I find that the image objects are always the last

**Comment by mara004** [comment, marked as answer]
Not sure what this is about, can you elaborate?
FWIW, I don't think the index in pdfium's array representation matters, it's the coordinates and matrix that is decisive for the appearance.
